data.sql

```diff
@@ -18,4 +18,50 @@ VALUES ('Pikachu', '01-07-2021', '1', 'false', '15.08');
 
 INSERT INTO animals 
 (name, date_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES ('Devimon', '05-12-2017', '5', 'true', '11');
\ No newline at end of file
+VALUES ('Devimon', '05-12-2017', '5', 'true', '11');
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Charmander', '02-08-2020', '0', 'false', '11');
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Plantmon', '11-15-2022', '2', 'true', '-5.7');
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Squirtle', '04-02-1993', '3', 'false', '-12.13');
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Angemon', '06-12-2005', '2', 'true', '-45');
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Boarmon', '06-07-2005', '7', 'true', '20.4');
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Blossom', '10-13-1998', '3', 'true', '17');
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species is NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-1-1';
+SAVEPOINT delete_one;
+UPDATE animals SET weight_kg = -1 * weight_kg;
+ROLLBACK TO delete_one;
+UPDATE animals SET weight_kg = -1 * weight_kg WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] <b>REQUIRED:</b> These should be moved to `queries.sql` as the project requires so, and Kindly make sure to verify that changes have been made before and after `COMMIT`. something like this:

```sql
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit

Reply:
moved to queries.sql